core/src/databases/table_schema.rs

```diff
@@ -208,6 +223,7 @@ impl TableSchema {
                         TableSchemaFieldType::Float => "REAL",
                         TableSchemaFieldType::Text => "TEXT",
                         TableSchemaFieldType::Bool => "BOOLEAN",
+                        TableSchemaFieldType::DateTime => "TEXT",
```

Comment:
What drove the decision of TEXT vs INT ? Mind sharing examples of queries for both?

```
SELECT * FROM issues WHERE "created is within the last 3 weeks"
```

When created is represented as TEXT datetime and when created is represented as INT epoch? :pray:

Reply:
The major reason is that if the model sees a query output where the date is just the raw value from DB, it can understand it. Model will have a hard time interpreting an integer

Reply:
Similarly it's easier for the model to generate a valid string timestamp for eg "3 days ago" if you give it the current date. Almost impossible with an integer

Reply:
Do you mean the last 3 weeks one ?
would look like this:
```
SELECT * FROM your_table_name
WHERE datetime(created) > datetime('now', '-3 weeks');
```

Reply:
IMO the support for ISO8601 is really good enough

Reply:
Just trying to understand the differences.

Reply:
Like this
```
SELECT * FROM your_table_name
WHERE created > strftime('%s', 'now', '-3 weeks');
```

Reply:
It's just a tiny bit worse

Reply:
Ok! Thanks for the details, agreed it seems like a clear winner :+1: